src/global/index.js

```diff
@@ -77,7 +77,7 @@ exports.before = {
 const after = {
 	// todo select is better but need more stable implementations
 	all: [iff(isProvider('external'),
-		filterOutResults('__v', 'createdAt', 'updatedAt')),
```

Comment:
Why change this? @CordlessWool

Reply:
Its a long time ago, I could not remember, but its filter out before sending it to the client? For the client the data are not necassery